packages/launchpad/src/checklist-item/index.tsx

```diff
@@ -53,7 +61,7 @@ const ChecklistItem = ( { task, isPrimaryAction }: { task: Task; isPrimaryAction
 				<Button
 					className="checklist-item__task-content"
 					data-task={ id }
-					onClick={ actionDispatch }
+					onClick={ onClick }
```

Comment:
This feels like an unnecessary change to the API for `ChecklistItem`, as all callers now need to specify `onClick`, whereas before they only needed to specify `task`. Might it be cleaner from an API perspective to use `task.actionDispatch()` as a fallback?
```suggestion
					onClick={ onClick ?? task.actionDispatch }
```

Reply:
Fixed here with tests [4a9fdab](https://github.com/Automattic/wp-calypso/pull/87141/commits/4a9fdab93fd805ab48cf7351eb0e58dc4a36c33d)